### PR TITLE
Init version 24.04.0-SNAPSHOT

### PR DESCRIPTION
**Pull request by NvTimLiu**
bump up JNI version to  24.04.0-SNAPSHOT

Signed-off-by: Tim Liu [timl@nvidia.com](mailto:timl@nvidia.com)

### CODE REVIEW

**Comment by NvTimLiu** [comment]
build

File: pom.xml
Comment (jlowe) [inline pom.xml:1]: 2024 copyrights

File: src/main/cpp/CMakeLists.txt
Comment (jlowe) [inline src/main/cpp/CMakeLists.txt:1]: 2024 copyrights

**Comment by NvTimLiu** [comment]
build

**Comment by NvTimLiu** [comment]
build

**Review by YanxuanLiu — APPROVED** [review]

**Review by yinqingh — APPROVED** [review]

**Pull request merged**